taxonomies/categories.txt

```diff
@@ -36644,10 +40755,17 @@ zh:家禽
 en:Poultries from Ancenis
 fr:Volaille d'Ancenis
 country:en:France
+protected_name_file_number_en: PGI-FR-0167
+#protected_name_file_number_en: PGI-FR-0167-AM02
+protected_name_type:en: pgi
+origins:en: en:France
 
 <en:Poultries
 en:Poultries from Bresse
-fr:Volaille de Bresse
+fr:Volaille de Bresse, Poulet de Bresse, Poularde de Bresse, Chapon de Bresse
```

Comment:
Poulet de Bresse, Poularde de Bresse, Chapon de Bresse  -> thouse should be children
no need to repeat the properties, they are inherited

Reply:
fixed. I kept Volailles de Bresse as parent